src/Tribe/Admin/Helpers.php

```diff
@@ -109,14 +109,14 @@ public function is_screen( $id = null ) {
 		}
 
 		// Match a specific page
-		if ( $current_screen->id === $id ) {
-			return true;
+		if ( ! is_null( $id ) && $current_screen->id !== $id ) {
+			return false;
 		}
 
 		// Match any post type page in the supported post types
 		$defaults = apply_filters( 'tribe_is_post_type_screen_post_types', Tribe__Main::get_post_types() );
-		if ( ! in_array( $current_screen->post_type, $defaults ) ) {
-			return false;
+		if ( in_array( $current_screen->post_type, $defaults ) ) {
```

Comment:
Seems like you can simplify this section of code by removing the next two returns and just returning the result of `in_array` as will return either `true/false` depending on your operation.

```php
return in_array( $current_screen->post_type, $defaults );
```